testing/i3blocks/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Marvin Steadfast <marvin@xsteadfastx.org>
+# Maintainer: Marvin Steadfast <marvin@xsteadfastx.org>
+pkgname=i3blocks
+pkgver=20190321_git
```

Comment:
```suggestion
pkgver=1.4
```

Reply:
but this is a really really old release... i thought its better to use the master branch... but i will do it this way... i checked the package on debian and its 1.4 too.